### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by tnorling**
- Add `msal-node-extensions` to issue templates
- Adds clarification that "msal" = v1 and "msal-browser" = v2 and moves msal-browser to the top

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -24,9 +24,9 @@ body:
   attributes:
     label: Core Library
     options: 
-      - "@azure/msal or msal"
-      - "@azure/msal-browser"
+      - "@azure/msal-browser (v2)"
       - "@azure/msal-node"
+      - "@azure/msal or msal (v1)"
```
Comment (jasonnutter) [inline .github/ISSUE_TEMPLATE/bug_report.yml:29]: I would if it is easier to have display names, e.g.

- MSAL.js v2 (@azure/msal-browser)
- MSAL.js v1 (@azure/msal or msal)
- MSAL React (@azure/msal-react)

etc

Comment (tnorling) [reply .github/ISSUE_TEMPLATE/bug_report.yml:29]: I like that suggestion

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -24,9 +24,9 @@ body:
...
     options: 
-      - "@azure/msal or msal"
-      - "@azure/msal-browser"
-      - "@azure/msal-node"
+      - "MSAL.js v2 (@azure/msal-browser)"
+      - "MSAL.js v1 (@azure/msal or msal)"
+      - "MSAL Node (@azure/msal-node)"
       - "passport-azure-ad"
       - "adal-node"
       - "adal.js"
```
Comment (jasonnutter) [inline .github/ISSUE_TEMPLATE/bug_report.yml:32]: These three can also use display names

**Review by jasonnutter — APPROVED** [review]

**Review by hectormmg — APPROVED** [review]
LGTM!

**Pull request merged**